DotnetRuntimeMetadata.json

```diff
@@ -1,13 +1,13 @@
 {
   "sdk": {
-    "channel": "7.0.1xx-preview2",
+    "channel": "7.0.1xx-preview3",
     "quality": "signed",
     "qualityFallback": "daily",
-    "packageVersionPattern": "7.0.0-preview.2",
+    "packageVersionPattern": "7.0.0-preview.3",
     "sdkImageVersion": "7.0.100",
-    "nextChannel": "7.0.1xx-preview2",
+    "nextChannel": "7.0.1xx-preview3",
     "azureFeed": "",
-    "sdkImageOverride": "7.0.100-preview.2.22153.17"
+    "sdkImageOverride": ""
```

Comment:
Why do we still need the same in global.json? Weird.

Reply:
This makes the GitHub workflow that updates the build stick to this specific version. If we keep this here, we will never get daily updates. The value is global.json is the one we are building against.